frontend/src/app/uploads/DeviceLookup/DeviceLookupContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  DeviceType,
+  useGetSpecimenTypesQuery,
+  useGetDeviceTypeListQuery,
+} from "../../../generated/graphql";
+import { LoadingCard } from "../../commonComponents/LoadingCard/LoadingCard";
+
+import DeviceLookup from "./DeviceLookup";
+
+const DeviceLookupContainer = () => {
+  const { data: specimenTypesResults } = useGetSpecimenTypesQuery({
+    fetchPolicy: "no-cache",
+  });
+
+  const { data: deviceTypeResults } = useGetDeviceTypeListQuery({
```

Comment:
These two queries are loading way more than we need (such as supported diseases (this will cause an n+1, we still havent fixed it), testLength)
https://github.com/CDCgov/prime-simplereport/blob/c5ce8eba7979349ad5611f2144a073041cab316c/frontend/src/generated/graphql.tsx#L5003-L5020

 also you can just load everything you need from DeviceType, so you can shrink it down to 1 query.
 
 so no need to run another query for all specimen types its already in the swabTypes of the DeviceType query,